### ISSUE TITLE
[a11y]: UI Shell [Header] - lists should better defined

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#15272]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.41.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.41.1

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Description
Two different list should be defined

Divider is provided indicating two different list. But screen reader is announcing as a single list.

### Recommended Fix
Two different list should be defined and screen reader should announce the list items count along with the list correctly.

People who uses screen reader will get confused with the structure of the page.

### WCAG 2.1 Violation
1.3.1 Info and Relationships

**Comment by kennylam** [comment]
This component has undergone major changes since this issue has been reported. Worth comparing the latest version to this issue to see if it's still a valid concern.
